### PR TITLE
Add Collatz Conjecture exercise.

### PR DESCRIPTION
**Pull request by KalebHawkins**
- Update config.json to include exercise slug.
  - Create the required files and directory structure for exercise.
  - Fill out `instructions.md`.
  - Configure `$ExerciseDir/.meta/config.json` file.
  - Add exercise stub.
  - Add exercise tests.
  - Add example solution.

Linting: Tested
    ./bin/configlet lint
    The lint command is under development.
    Please re-run this command regularly to see if your track passes the latest linting rules.

    Basic linting finished successfully:
    - config.json exists and is valid JSON
    - config.json has these valid fields:
        language, slug, active, blurb, version, status, online_editor, key_features, tags
    - Every concept has the required .md files
    - Every concept has a valid links.json file
    - Every concept has a valid .meta/config.json file
    - Every concept exercise has the required .md files
    - Every concept exercise has a valid .meta/config.json file
    - Every practice exercise has the required .md files
    - Every practice exercise has a valid .meta/config.json file
    - Required track docs are present
    - Required shared exercise docs are present

Pester: Tested
    ./bin/run-tests.ps1

    ```
    Starting discovery in 1 files.
    Discovery found 6 tests in 5ms.
    Running tests.
    [+] /tmp/collatz-conjecture/collatzConjecture.tests.ps1 38ms (16ms|17ms)
    Tests completed in 39ms
    Tests Passed: 6, Failed: 0, Skipped: 0 NotRun: 0
    ```

### CODE REVIEW

File: exercises/practice/collatz-conjecture/.meta/config.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "files": {
+    "solution": [
+      "collatzConjecture.ps1"
+    ],
+    "test": [
+      "collatzConjecture.tests.ps1"
+    ],
+    "example": [
+      ".meta/collatzConjecture.example.ps1"
+    ]
```
Comment (ErikSchierboom) [inline exercises/practice/collatz-conjecture/.meta/config.json]: All the existing files use PascalCase for their files. That's also how other .NET platform languages define their file name, so I think it makes sense to use that here too. Unless I'm missing a specific convention?

**Review by ErikSchierboom — COMMENTED** [review]
Great! Just one comment.

File: exercises/practice/collatz-conjecture/.meta/config.json (hunk 1 above)
Comment (KalebHawkins) [reply exercises/practice/collatz-conjecture/.meta/config.json]: No that is right. I will correct this and re-push..

**Comment by KalebHawkins** [comment]
@ErikSchierboom Not going to lie.. I didn't mean to close this but I cannot reopen it for some reason... I think I may have done something wrong when I tried to squash my commits.. [PR 170](https://github.com/exercism/powershell/pull/169) was created in its place.

**Comment by ErikSchierboom** [comment]
Weird. I'll check the other PR.

**Pull request closed without merging**